sysinfo.go

```diff
@@ -32,13 +35,51 @@ const (
 	ProcessorArchitectureAMD64   ProcessorArchitecture = wrappers.PROCESSOR_ARCHITECTURE_AMD64
 )
 
+type DisplayDevice struct {
+	DeviceName   string
+	DeviceString string
+	StateFlags   uint32
+	DeviceID     string
+	DeviceKey    string
+}
+
+type DisplayMonitorInfo struct {
+	Handle wrappers.HMONITOR
+	Hdc    wrappers.HDC
+	Rect   wrappers.RECT
+}
+
 type ProcessorInfo struct {
 	ProcessorArchitecture ProcessorArchitecture
 	NumberOfProcessors    uint
 	ProcessorLevel        uint
 	ProcessorRevision     uint
 }
 
+func EnumAllDisplayDevices() []DisplayDevice {
```

Comment:
Since we're just returning all of the devices rather than requiring users to enumerate through them, let's just call this `GetAllDisplayDevices`.